docs/docs/behaviors/hold-tap.md

```diff
@@ -30,11 +33,11 @@ We call this the 'hold-preferred' flavor of hold-taps. While this flavor may wor
 
 When the hold-tap key is released and the hold behavior has not been triggered, the tap behavior will trigger.
 
-![Hold-tap comparison](../assets/hold-tap/comparison.png)
+![Hold-tap comparison](../assets/hold-tap/comparison.svg)
 
 ### Basic usage
 
-For basic usage, please see [mod-tap](mod-tap.md) and [layer-tap](layers.md) pages.
+For basic usage, please see the [mod-tap](mod-tap.md) and [layer-tap](layers.md) pages.
```

Comment:
```suggestion
For basic usage, please see the [mod-tap](mod-tap.md) and [layer-tap](layers.md#layer-tap) pages.
```
While we are touching here, might as well link to the section (assuming my link works?)